Cargo.lock

```diff
@@ -23,9 +23,9 @@ dependencies = [
 
 [[package]]
 name = "adler2"
-version = "2.0.1"
+version = "2.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
I think the changes to `Cargo.lock` and `public_api.txt` are unrelated? should they be reverted?